jsk_2015_05_baxter_apc/WorkOrder[] array